### ISSUE TITLE
[A11y] Work Show Page-Multiple - Image alt attribute

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1990]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n

Child object images, "Is part of" images, and "related items" images lack the `alt` attribute. These images are the only elements present inside a link (`<a>`), and therefore must have a non-null `alt` attribute.

### Suggested Fix

Make the images' `alt` attribute reflect the purpose of the image, which is to link to another page. Typically this means having the accessible name reflect the destination page's title.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Check the alt attributes for all images on a compound parent and compound child object. Make sure the alt attribute reflects the purpose of the image.
